ibis/expr/operations/generic.py

```diff
@@ -273,6 +273,20 @@ class HashBytes(Value):
     shape = rlz.shape_like("arg")
 
 
+@public
+class HashHexDigest(Value):
+    arg: Value[dt.String | dt.Binary]
```

Comment:
Any chance we can call it `HexDigest`?

Reply:
It is done.

Reply:
Can we reuse the `how` annotation for both `HashBytes` and `HashDigest`?

Reply:
Hey @kszucs -- I would like to do that eventually but not here.  There's a fair bit of inconsistency in the output type of various hashing functions between the backends and I'd like to address that in `the-epic-split` instead of here.

Reply:
Sure, it is not that important.